tensorflow/lite/micro/tools/make/targets/esp32_makefile.inc

```diff
@@ -1,7 +0,0 @@
-# Settings for Espressif ESP32
-
-ifeq ($(TARGET), esp)
```

Comment:
No need for this check! As the file is picked only if it is esp build.

Removed this in renamed makefile `esp_makefile.inc`.